src/app/loadout-builder/generated-sets/mod-utils.ts

```diff
@@ -185,6 +185,10 @@ export function canSetTakeGeneralAndSeasonalMods(
 ) {
   const assignments: Record<number, LockedArmor2Mod[]> = {};
 
+  for (const item of set) {
+    assignments[item.hash] = new Array<LockedArmor2Mod>();
```

Comment:
Instead of `new Array` just do `[]`

Reply:
I swear it didn't like that when I initial did this a couple of months ago. Fixed that and another line that got deleted at some point.